packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -6,20 +6,23 @@
           "value": "0 auto 15px auto"
         },
         "text": {
+          "color": {
```

Comment:
and this one would be `fgColor`

Reply:
Thanks! 947c395b5ba093459009c0c08adef9f67a31c797